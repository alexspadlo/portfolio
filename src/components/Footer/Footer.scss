@import "../../stylesheet/_vars.scss";
