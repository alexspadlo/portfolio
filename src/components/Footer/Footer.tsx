import React from 'react'
import s from './Footer.scss'

const Footer: React.FC = () => {
    return (
        <div className='container-fluid'>
            <div className="row pb-2">
                <div className="col-11 p-0 m-auto text-center">
                    <hr className={s.goldHR} />
                    <p className={`${s.fontBold} ${s.txtcolWhite}`}>Copyright Â© {new Date().getFullYear()} Aleksandra Spadlo</p>
                </div>
            </div>
        </div>

    )
}

export default Footer