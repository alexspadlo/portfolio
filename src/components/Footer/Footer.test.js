import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'

describe('Test on <Footer /> component', () => {
    test('Should render without the props', () => {
        const componentNoprops = renderer.create(<Footer />)
        const treeNoprops = componentNoprops.toJSON()
        expect(treeNoprops).toMatchSnapshot()
    })
})
