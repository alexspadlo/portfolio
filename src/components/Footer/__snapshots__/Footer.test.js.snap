// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <Footer /> component Should render without the props 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row pb-2"
  >
    <div
      className="col-11 p-0 m-auto text-center"
    >
      <hr
        className="goldHR"
      />
      <p
        className="fontBold txtcolWhite"
      >
        Copyright Â© 
        2023
         Aleksandra Spadlo
      </p>
    </div>
  </div>
</div>
`;
